Umbraco-Cloud/Set-Up/Examine-Configuration/index.md

```diff
@@ -5,7 +5,8 @@ The indexes are used in order to provide search capability in the backoffice (no
 That means that once a media item is stored, metadata about the media item will be stored in the index, and fetched from it when requested.
 
 These indexes need to be kept in sync, and are required for Umbraco to start up. The way this is done, will directly affect your sites performance.
-When working on Umbraco Cloud, there is some key settings that changes, dependending on whether  you are working locally or on your development/staging/live environment. 
+When working on Umbraco Cloud, there is some key settings that changes, depending on whether you are working locally or on your development/staging/live environment. 
```

Comment:
This needs to be completely removed. Cloud handles all of this by transforms, the customer doesn't need to do anything